llvm-runtimes/compiler-rt-sanitizers/compiler-rt-sanitizers-22.0.0.9999.ebuild

```
@@ -115,9 +118,20 @@ src_configure() {
 	BUILD_DIR=${WORKDIR}/compiler-rt_build
 
 	if use clang; then
-		local -x CC=${CHOST}-clang
-		local -x CXX=${CHOST}-clang++
+		local -x CC=${CHOST}-clang-${LLVM_MAJOR}
+		local -x CXX=${CHOST}-clang++-${LLVM_MAJOR}
 		strip-unsupported-flags
+
+		# The full clang configuration might not be ready yet. Use the partial
+		# configuration files that are guaranteed to exist even during initial
+		# installations and upgrades.
+		local flags=(
+			--no-default-config
+			--config=${ESYSROOT}/etc/clang/${LLVM_MAJOR}/gentoo-{rtlib,stdlib,linker}.cfg
```

```suggestion
			--config="${ESYSROOT}"/etc/clang/${LLVM_MAJOR}/gentoo-{rtlib,stdlib,linker}.cfg
```

While indeed it gets flattened below, there's no harm in doing it right *and* it gives us less warnings.